src/python/hangman/README.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Lærerveiledning - Hangman
+level: 3
+language: nb
+---
+
+
+# Om oppgaven {.activity}
+
+I denne oppgaven skal vi lage det klassiske spillet Hangman. Der brukeren prøver å gjette seg frem til et hemmelig ord. 
+
+
+## Oppgaven passer til: {.check}
+
+ __Fag__: Programmering
```

Comment:
får vi knyttet denne til språk-fag? ikke noe it fra vgs? om ikke noen av delene: ignorer denne kommentaren.